web/src/app/help/help.component.ts

```diff
@@ -1,12 +1,14 @@
 import {Component, OnInit} from '@angular/core';
+// import {MatButtonModule} from '@angular/material/button';
 
 @Component({
   selector: 'app-help',
   templateUrl: './help.component.html',
-  styleUrls: ['./help.component.scss']
+  styleUrls: ['./help.component.scss'],
+  preserveWhitespaces: true
```

Comment:
why is this needed?

Reply:
No, it's not. I was trying to find a way to keep the whitespace but this is actually not the problem.